public/apps/configuration/panels/permission-list/permission-list.tsx

```diff
@@ -367,44 +398,56 @@ export function PermissionList(props: AppDependencies) {
+        count={permissionList.length}
+      />
       {loading ? (
         <EuiLoadingContent />
       ) : accessErrorFlag ? (
         <AccessErrorComponent loading={loading} dataSourceLabel={dataSource && dataSource.label} />
       ) : (
         <EuiPageContent>
-          <EuiPageContentHeader>
-            <EuiPageContentHeaderSection>
-              <EuiTitle size="s">
-                <h3>
-                  Permissions
-                  <span className="panel-header-count">
-                    {' '}
-                    ({Query.execute(query || '', permissionList).length})
-                  </span>
-                </h3>
-              </EuiTitle>
-              <EuiText size="xs" color="subdued">
-                Permissions are individual actions, such as cluster:admin/snapshot/restore, which
-                lets you restore snapshots. Action groups are reusable collections of permissions,
-                such as MANAGE_SNAPSHOTS, which lets you view, take, delete, and restore snapshots.
-                You can often meet your security needs using the default action groups, but you
-                might find it convenient to create your own.{' '}
-                <ExternalLink href={DocLinks.PermissionsDoc} />
-              </EuiText>
-            </EuiPageContentHeaderSection>
-            <EuiPageContentHeaderSection>
-              <EuiFlexGroup>
-                <EuiFlexItem>{actionsMenu}</EuiFlexItem>
-                <EuiFlexItem>{createActionGroupMenu}</EuiFlexItem>
-              </EuiFlexGroup>
-            </EuiPageContentHeaderSection>
-          </EuiPageContentHeader>
+          {useUpdatedUX ? null : (
+            <EuiPageContentHeader>
+              <EuiPageContentHeaderSection>
+                <EuiTitle size="s">
+                  <h3>
+                    Permissions
+                    <span className="panel-header-count">
+                      {' '}
+                      ({Query.execute(query || '', permissionList).length})
+                    </span>
+                  </h3>
+                </EuiTitle>
+                <EuiText size="xs" color="subdued">
+                  Permissions are individual actions, such as cluster:admin/snapshot/restore, which
+                  lets you restore snapshots. Action groups are reusable collections of permissions,
+                  such as MANAGE_SNAPSHOTS, which lets you view, take, delete, and restore
+                  snapshots. You can often meet your security needs using the default action groups,
+                  but you might find it convenient to create your own.{' '}
+                  <ExternalLink href={DocLinks.PermissionsDoc} />
+                </EuiText>
+              </EuiPageContentHeaderSection>
+              <EuiPageContentHeaderSection>
+                <EuiFlexGroup>
+                  <EuiFlexItem>{actionsMenu}</EuiFlexItem>
+                  <EuiFlexItem>{createActionGroupMenu}</EuiFlexItem>
```

Comment:
Nit: It may be possible to see if Oui components are available.

Reply:
OSD aliases OUI as Eui, so we are good on that front.